### ISSUE TITLE
No way to create slice objects

### ISSUE THREAD

**Issue by halostatue** [halostatue/rubypython#25]
I tried to slice a numpy array but this doesn't work since Ruby doesn't understand Python's slice syntax. If I could create a slice object using the Python built-in slice function, I could get around this by passing a slice object to Numpy's getslice method, but afaik there's no way to access this.

Perhaps RubyPython could provide a built-in for this? Access to other Python built-ins would be nice as well, though less necessary. Specifically, access to len and type would be handy.

**Comment by lokeshh** [comment]
Yes this would be very useful. I'm able to run the majority of numpy except indexing because of this issue.

**Comment by lokeshh** [comment]
One way I can think it might work is by import the function `slice` from python somehow. Any idea how to do it?

**Comment by lokeshh** [comment]
I managed to create one. Here's how:
```rb
RubyPython::PyMainClass.send(:new).slice(2, 3)

```
